db/doctrine_migrations/Version20230227190006.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20230227190006 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Toevoegingen voor web-push';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql('CREATE TABLE push_abonnement (id INT AUTO_INCREMENT NOT NULL, uid VARCHAR(4) COMMENT \'(DC2Type:uid)\' NOT NULL, client_endpoint VARCHAR(255) NOT NULL, client_keys VARCHAR(255) NOT NULL, INDEX IDX_BVRODGMKX13C5GFL (uid), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_general_ci` ENGINE = InnoDB');
```

Comment:
Doctrine zou `IDX_BVRODGMKX13C5GFL` droppen

Reply:
Hmm ik snap eigenlijk het hele probleem niet haha. Ik heb niet veel ervaring met dit soort SQL migraties. Ik heb het van `Version20220525195704.php` gekopieerd, dus daar had ik ook die alter met FK vandaan. Is er iets wat ik daarvoor nog had moeten toevoegen ergens anders?

Reply:
```suggestion
        $this->addSql('CREATE TABLE push_abonnement (id INT AUTO_INCREMENT NOT NULL, uid VARCHAR(4) COMMENT \'(DC2Type:uid)\' NOT NULL, client_endpoint VARCHAR(255) NOT NULL, client_keys VARCHAR(255) NOT NULL, PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_general_ci` ENGINE = InnoDB');
```

Dit volgens mij